solidity/contracts/examples/CallbackContract.sol

```diff
@@ -9,19 +9,12 @@ contract CallbackContract {
 
     uint256 internal _lastEntry;
 
-    /**
-     * @dev Example of a callback method. Method signature can be
-     * calculated as bytes4(keccak256("callback(uint256)")
-    */
-    function callback(uint256 requestResponse)
+    function __beaconCallback(uint256 entry)
         public
```

Comment:
do we want to have `public` in a new line?

Reply:
This is how our Solidity linters work in `keep-ecdsa`, see [here](https://github.com/keep-network/keep-ecdsa/blob/master/solidity/contracts/BondedECDSAKeepFactory.sol). I don't think we ported this work to `keep-core` already but we'll be doing it soon. And then we'll have less work here.

Reply:
hah.. I see that it is kinda random in `keep-ecdsa`. I guess wip :) anyway, not biggy, we can leave it as is then

Reply:
> hah.. I see that it is kinda random in keep-ecdsa

Yep. We should all have pre-commit hooks installed with the same linting configuration but we want to make it enforced on the build level.